### PR TITLE
docs(localpv): Add note about k0s use of custom kubelet directory

### PR DESCRIPTION
**Pull request by t3hmrman**
## Pull Request template

Please, go through these steps before you submit a PR.

**Why is this PR required? What issue does it fix?**:

`k0s`'s alternate `/var/lib/kubelet` requires a change to the operator YAML,

**What this PR does?**:

Add a note about `k0s`'s required `/var/lib/kubelet` change, similar to `microk8s`.

**Does this PR require any upgrade changes?**:

No

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Any additional information for your reviewer?** :

**Checklist:**
- [ ] Fixes #<issue number>
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,12 @@ $ kubectl apply -f https://openebs.github.io/charts/zfs-operator.yaml
 ```
 
 We can also install it via kustomize using `kubectl apply -k deploy/yamls`, check the [kustomize yaml](deploy/yamls/kustomization.yaml).
-For microk8s, we need to change the kubelet directory to `/var/snap/microk8s/common/var/lib/kubelet/`, we need to replace `/var/lib/kubelet/` with `/var/snap/microk8s/common/var/lib/kubelet/` at all the places in the operator yaml and then we can apply it on microk8s.
+
+**NOTE** For some Kubernetes distributions, the `kubelet` directory must be changed at all relevant places in the YAML powering the operator (both the `openebs-zfs-controller` and `openebs-zfs-node`).
```
Comment (pawanpraka1) [inline README.md:98]: Add ":" here, make it **Note:**.

**Review by pawanpraka1 — COMMENTED** [review]
Thanks for this PR @t3hmrman.

File: README.md (hunk 1 above)
Comment (t3hmrman) [reply README.md:98]: Will do!

- [ ] add `:` to the "Note"

**Review by pawanpraka1 — APPROVED** [review]
Thanks @t3hmrman.

**Pull request merged**